scripts/us_epa/parent_company/testdata/expected/table/EpaParentCompanyTable.csv

```diff
@@ -0,0 +1,9 @@
+dcid,parentCompanyName,address,locatedIn
+dcid:EpaParentCompany/NationalFuelGasCompany,National Fuel Gas Company,"6363 Main Street, Williamsville, Ny - 14221","dcid:zip/14221, dcid:geoId/36029"
+dcid:EpaParentCompany/UsGovernment,Us Government,,dcid:zip/

```

Comment:
we shouldn't emit "zip/" -- in these cases, can we emit blank?

Reply:
Done